mobile/status.go

```diff
@@ -1071,6 +1072,17 @@ func writeHeapProfile(dataDir string) string { //nolint: deadcode
 	return makeJSONResponse(err)
 }
 
+// StartMutexProfiling starts mutex profiling and HTTP server for pprof
+func StartMutexProfiling(address string) string {
```

Comment:
Perhaps just keep it generic `StartProfiling`. It's not only starting the mutex thing.